bodhi/tests/client/test___init__.py

```diff
@@ -721,7 +721,7 @@ def test_close_bugs_flag(self, send_request):
                 return_value=client_test_data.EXAMPLE_UPDATE_MUNCH, autospec=True)
     def test_display_name_flag(self, send_request):
         """
-        Assert correct behavior with the --close-bugs flag.
+        Assert correct behavior with the --display-name flag.
```

Comment:
This change is not related to this commit. Let's add second commit with this correction.